### ISSUE TITLE
Error Expecting value: line 1 column 1 (char 0)

### ISSUE THREAD

**Issue by fellnerse** [microsoft/sample-app-aoai-chatGPT#122]
At the moment with the latest commit (de20819906df124a512de896c9cea209e00d383f), the conversation always ends with a timeout I assume (after waiting for half a minute for an answer):
```bash
Error
Expecting value: line 1 column 1 (char 0)
```
I guess that error happens due to trying to parse something as json which is not json. 
The error sometimes happens right after the first message, sometimes I have more time (messages).

**Comment by pamelafox** [comment]
Please look for the full logs to see more details about the error. Open the App Service app in the Portal, find Advanced Tools in the left hand nav, open that tools website up, then click "Download as zip" on that page. When the zip downloads, open it up and find the logs named "*_default_docker.txt". Scroll down in those to find the latest logs.

**Comment by KonTheCat** [comment]
Hi! I also have this issue. The logs from *_default_docker.txt do not show anything interesting, nor does the log stream from the same source. Would greatly appreciate any assistance.

**Comment by davidm00** [comment]
Hi are you still experiencing this error? @KonTheCat @fellnerse

**Comment by KonTheCat** [comment]
Yes. This is still an issue. A contact at MS says a fix is expected around EOM.

**Comment by muhammadharoon110** [comment]
Hi @KonTheCat, Are you still hvaing this error?   I have deployedd a chatbot on webapp.OAI service with private endpoint. But when I am enabling private endpoint for cognitive search, I getting the same error. Somehow the bot on chatbot stop working. Any guidance in this regards? Thanks

**Comment by KonTheCat** [comment]
Hi @muhammadharoon110 - this is the form to apply for the relevant preview functionality: https://aka.ms/applyacsvpnaoaioyd
I got denied for this initially but my MS rep intervened and we got it. The feedback I got as a part of this process is that Microsoft is working on resolving this sooner rather than later. I got the impression that this is a priority for them.
